### ISSUE TITLE
Windows Minidump Segfault

### ISSUE THREAD

**Issue by Maijin** [radare/radare2#3649]
http://pastie.org/private/wudcb6nx5bgusspzznzq

The file: http://rghost.net/private/7vRjfNr8Q/42b2b3be50f73dce291d2e9a8e1fe101

**Comment by Maijin** [comment]
@davispuh  You may be interested

**Comment by davispuh** [comment]
I don't really have time to look into it, but that crash happens at [libr/bin/p/bin_mdmp.c#94](/radare/radare2/blob/master/libr/bin/p/bin_mdmp.c#L94)

It might be related to radare API changes since I wrote it then, there it doesn't even touch mdmp object at all. Maybe `RBinInfo, ret->file` needs to be allocated before.

**Comment by davispuh** [comment]
yeah it is so, will need fixing all accesses to allocate it. It was introduced with this commit dff11411ccacacd167e7589f7a7840ef80712b60 but I wrote it way way earlier :D

**Comment by radare** [comment]
The segfault is gone, but there are some invalid sizes in sections:

```
$ r2  bins/mdmp/jusched.dmp 
ss 3
ss 17
ss 4
ss 5
ss 7
ss 15
ss 12
ss 0
ss 0
ss 0
Invalid size (0xfeef04bd) for section '_1' at 0x00000408
Invalid size (0xfe773020) for section '_53' at 0x00027748
Invalid size (0xfeef04bd) for section '_1' at 0x00000408
Invalid size (0xfe773020) for section '_53' at 0x00027748
Invalid size (0xfeef04bd) for section '_1' at 0x00000408
Invalid size (0xfe773020) for section '_53' at 0x00027748
Invalid size (0xfeef04bd) for section '_1' at 0x00000408
Invalid size (0xfe773020) for section '_53' at 0x00027748
Invalid size (0xfeef04bd) for section '_1' at 0x00000408
Invalid size (0xfe773020) for section '_53' at 0x00027748
 -- Pass '-j' to rabin2 to get the information of the binary in JSON format.
[0x00000000]> pd 20

```

**Comment by radare** [comment]
I'm moving minidump and msil support to radare2-extras. Sorry, there's no time to review and fix this before the release. Can you take a look?

**Comment by davispuh** [comment]
it's fine, just fixed that, see radare/radare2-extras#24, but overall minidump parsing is still very incomplete.
